jdaviz/configs/mosviz/plugins/slit_overlay/slit_overlay.py

```diff
@@ -13,7 +13,35 @@
 
 import bqplot
 
-__all__ = ['SlitOverlay']
+__all__ = ['SlitOverlay', 'jwst_header_to_skyregion']
+
+
+def jwst_header_to_skyregion(header):
+    s_region = header['S_REGION']
+    footprint = s_region.split("POLYGON ICRS")[1].split()
+    ra = np.array(footprint[::2], dtype=np.float)
+    dec = np.array(footprint[1::2], dtype=np.float)
```

Comment:
Where did this algorithm come from? Are you sure this is correct?

What I dug up over at https://github.com/spacetelescope/jwst/blob/02ca26dd2fcaaf655144949eaf579db2e8d07d11/jwst/assign_wcs/util.py#L922 implies you might need GWCS. Best to confirm with `jwst` people if you are not sure.

Reply:
That's a question for @javerbukh

Reply:
Since this is now separated from UI stuff, this also means we can write unit tests for it. 😉